tests/server/auth/test_sqlalchemy_store.py

```diff
@@ -424,3 +425,29 @@ def test_delete_registered_model_permission(store):
     ) as exception_context:
         store.get_registered_model_permission(name1, username1)
     assert exception_context.value.error_code == ErrorCode.Name(RESOURCE_DOES_NOT_EXIST)
+
+
+def test_rename_registered_model_permission(store):
+    # create 2 users and create 2 permission for the model registry with the same name
+    model_registry_name = random_str()
```

Comment:
```suggestion
    model_name = random_str()
```
It's `registered model's name`, not model registry